### ISSUE TITLE
[EN][Talent][Paladin] Eye for an Eye

### ISSUE THREAD

**Issue by purabasura666** [Atlantiss/BugTracker#9573]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: The talent Eye for an Eye in the Paladin's Retribution tree is bugged. Specifically, 2/2 Eye for an Eye works as intended. HOWEVER, 1/2 Eye for an Eye is NOT working at all.

**Current behaviour**: See description above.

**Expected behaviour**: See description above.

**Steps to reproduce the problem**:

1. See description above. Run three dungeons with 2/2 Eye for an Eye, in all cases the damage meter Recount registers damage from Eye for an Eye. 
2. Now with only one talent point, 1/2, run three dungeons again. Recount registers no damage at all from Eye for an Eye.
3. 

**Revision**: ACore rev. 5510

**Comment by Psychobird7** [comment]
Which specific dungeons did you do in your test runs?

**Comment by Buggingtrack** [comment]
I can confirm this very specific bug, always been running prot paladin with subspec retribution 1/2 eye for an eye, 1/2 improved judgement and ofc 3/3 crusade, and i noticed in my meters eye for an eye damage NEVER procced in fights like Grand Vizier Ertan (which spams lightning bolts, and most adds before this boss cast lots of spells), or even in raids like Bastion of Twilight or Firelands (with some fights having a polarity on offensive spells on tanks)

**Comment by Psychobird7** [comment]
Interesting bug.     

You should never run E4E as prot though. It's terrible.

**Comment by Buggingtrack** [comment]
@Psychobird7 honestly it is a good little dmg output since as a tank you don't really need a range upgrade on a spell if you are gonna use it on your target that you are tanking on melee range...
got a screenshot of proof on the fight i mentioned first
![wowscrnshot_093017_184849](https://user-images.githubusercontent.com/32219149/31047880-ca7db254-a612-11e7-8b99-070cc035e8af.jpg)

**Comment by purabasura666** [comment]
Thanks for your confirmation, Buggingtrack!
Yeah, 1/2 Eye for an Eye is definitely bugged, as it never ever procs.
As for 2/2, as retribution, it represents an average damage increase of 1.2% in heroic dungeons. Less than I would have guessed.

**Comment by purabasura666** [comment]
One quick question. How long has it been bugged? If it's something really recent, I'll maintain my spec and hope for a fix soon. Otherwise, I might just respec out of it.

**Comment by Buggingtrack** [comment]
It was (and still is) always been bugged tbh, never felt it procced as 1/2 when i was leveling the same character at around level 77, since i wanted to getting used to the only 10 yards bonus of the judgement instead of 20, which i thought it was way too unnecessary in PvE. Can just hope for a label and a fast fix on this little bug

**Comment by purabasura666** [comment]
When did you first notice it, Buggingtrack? I just leveled my paladin recently and changed from 2/2 to 1/2 a few days ago.

**Comment by Buggingtrack** [comment]
@purabasura666 when i was level 77 with my character, i guess? xD
It has been almost a month i had the feeling the talent 1/2 was not proccing at all

**Comment by purabasura666** [comment]
Yeah. The devs have been working really hard on the 4.3.4 content! They have done such a fabulous job! This bug, even though it affects a talent of a popular spec, might take a while to fix. No biggie. I think I'll respec out of it in the meantime.
